TestAssets/TestPackages/dotnet-portable-v1/dotnet-portable-v1.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We have a dotnet-back-compat.Tests test project that has tests for a 10 project running a 1.0 tool. The way we did, we can specify the TFM of the tool, so that it is easy to continue to add these tests. Would that tool replace this one here?